rllib/core/models/configs.py

```diff
@@ -104,8 +103,8 @@ class MLPHeadConfig(ModelConfig):
     hidden_layer_activation: str = "relu"
     output_activation: str = "linear"
 
-    @_framework_implemented()
-    def build(self, framework: str = "torch") -> Model:
+    def _validate(self, framework: str = "torch"):
```

Comment:
This validation should be performed by both, the free log std model and the standard model.